root/vote/VotingIndex.js

```diff
@@ -0,0 +1,465 @@
+/*
+ * @flow
+ * Copyright (C) 2019 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import * as React from 'react';
+
+import {withCatalystContext} from '../context';
+import Layout from '../layout';
+
+type VotingGuideRowProps = {
+  +guideName: string,
+  +mainUrl: string,
+  +subscribedArtistsUrl?: string,
```

Comment:
That might be replaced with a flag, as the URL for subscribed artists can be generated by appending `&conditions.9.field=artist&conditions.9.operator=subscribed` to the main URL.